### DISCUSSION TITLE
Running Inviscid Simulation

### DISCUSSION THREAD

**Discussion by Mohsin-Bashir** [mdolab/dafoam#607] [General]
Has anyone tried running inviscid simulation? 

Following dafoam workshop 2021 [1:32:04](https://www.youtube.com/watch?v=OIZd3I9UfGM&t=5524s), tried to run it for OneraM6 tutorial case. 
I changed `RASModel dummy` and `turbulence off` in [turbulenceProperties.txt](https://github.com/mdolab/dafoam/files/14691732/turbulenceProperties.txt) file and `mu 0` in [thermophysicalProperties.txt](https://github.com/mdolab/dafoam/files/14691722/thermophysicalProperties.txt) file.

But while running primal solution, it gives this error **pyDAFoam Error: Primal solution failed for the baseline design!**  
[logOpt.txt](https://github.com/mdolab/dafoam/files/14691767/logOpt.txt)

How can I run inviscid simulation?

**Comment by friedenhe** [comment]
It seems that the way you set up the inviscid case is correct. I am not sure why the case did not converge well. We have never run DAFoam for inviscid cases before, so you may need to explore how to converge the case...
